### PR TITLE
Add new text-black and text-white and use the for text color helper

### LINKED ISSUE #415: Use text-black and text-white for all text.
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently we use text-dark and text-light.

**What you expected to happen**:
Use text-black and text-white instead to decouple colorization of light and dark colors from impacting readability.

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Add `text-black/white` and use them for default text colors. Black and white colors are set to the default values of `dark` and `light` to keep the design change to a minimum.

**Which issue this PR fixes**:
fixes #415

### CODE REVIEW

**Comment by stp-ip** [comment]
Currently we still can "break" designs. Wanted to do it earlier than later. So your point would be that black and white don't work? Or did you just not wanna change the current working of the design? Happy to discuss this in basecamp in more detail.

File: assets/styles/_helpers.scss [hunk 1]
```diff
@@ -15,3 +15,11 @@
     display: none;
   }
 }
+
+.text-black {
```
Comment (stp-ip) [inline assets/styles/_helpers.scss]: Why is that needed? Bootstrap should have text-white and text-black by default...

File: layouts/partials/helpers/text-color.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {{- $bg := .self.Scratch.Get "bg" }}
-{{- $text_dark := cond (isset . "dark") (cond (eq .dark "") "" (printf "text-%s" .dark)) "text-dark" -}}
-{{- $text_light := cond (isset . "light") (cond (eq .light "") "" (printf "text-%s" .light)) "text-light" -}}
+{{- $text_black := cond (isset . "dark") (cond (eq .dark "") "" (printf "text-%s" .dark)) "text-black" -}}
```
[the remark this answers is not in the document]
Comment (mpourismaiel) [reply layouts/partials/helpers/text-color.html]: It's just fantastic :))

**Comment by stp-ip** [comment]
The other option would be `test-body`, which is closer to black than `black-50`

**Comment by mpourismaiel** [comment]
Good call. Changed to `.text-body`.

**Comment by stp-ip** [comment]
Merging now. This gives us a good base I think to also work on general coloring etc.
